usng.js

```diff
@@ -265,6 +310,35 @@
             return result;
         },
 
+        serializeUTM({zoneNumber, easting, northing}) {
+            return `${zoneNumber} ${easting}mE ${northing}mN` 
+        },
+
+        deserializeUTM(str) {
+            let zoneNumber, easting, northing
+
+            const processInvalidUTM = () => {
+                throw new Error( `Invalid UPS String: ${str}`)
+            }
+
+            const regExp = /^\s*(\d+)\s+(\d+)[mM][eE]\s+(\d+)[mM][nN]\s*$/
+            try {
+                [, zoneNumber, easting, northing] = regExp.exec(str)
+            } catch(err) {
```

Comment:
❓ Does a failure to match against the regex throw an error or would it just leave the contents of the array empty. In that case, would `zoneNumber < 1` or would it test out as something else?

Reply:
It throws an error.